front/app/containers/Admin/messaging/texting/PreviewMessage/index.tsx

```diff
@@ -0,0 +1,305 @@
+  box-sizing: border-box;
+  border-radius: 33px;
+  position: relative;
+`;
+
+const PhoneBezel = styled.div`
+  position: absolute;
+  width: 150px;
+  left: 53px;
+  height: 25.39px;
+  top: 0px;
+  background-color: black;
+`;
+
+const MessagesContainer = styled.div`
+  width: 210px;
+  margin: 0 auto;
+  margin-top: 30px;
+`;
+
+const PhoneMessage = styled.div`
+  & {
+    background-color: #eee;
+    position: relative;
+    border-radius: 20px;
+    padding: 8px 15px;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    text-align: left;
+    display: inline-block;
+    font-size: 0.9rem;
+  }
+  &:before {
+    content: '';
+    position: absolute;
+    z-index: 0;
+    bottom: 0;
+    left: -7px;
+    height: 20px;
+    width: 20px;
+    background: #eee;
+    border-bottom-right-radius: 15px;
+  }
+  &:after {
+    content: '';
+    position: absolute;
+    z-index: 1;
+    bottom: 0;
+    left: -10px;
+    width: 10px;
+    height: 20px;
+    background: white;
+    border-bottom-right-radius: 10px;
+  }
+`;
+
+const TextMessagePreview = (props: WithRouterProps) => {
+  const [showConfirmationModal, setShowConfirmationModal] = useState(false);
+  const [showDeleteTextModal, setShowDeleteTextModal] = useState(false);
+
+  const { campaignId } = props.params;
+  const campaign = useTextingCampaign(campaignId);
+
+  const confirmSendTextingCampaign = async () => {
+    console.log('disable send button here');
+    try {
+      console.log('implement send BE call here');
+      console.log(
+        'if successful, redirect to the view page for the newly created draft message'
+      );
+    } catch (e) {
+      console.log('fail', e);
+    }
+  };
+
+  const confirmDeleteTextingCampaign = async () => {
+    try {
+      const result = await deleteTextingCampaign(campaignId);
+      console.log('successful delete', result);
+      const url = `/admin/messaging/texting`;
+      clHistory.replace(url);
+    } catch (e) {
+      console.log('fail', e);
+    }
+  };
+
+  // actual error state when campaign not found
+  if (isNilOrError(campaign)) return null;
+
+  const { message, phone_numbers } = campaign.attributes;
+
+  return (
+    <>
+      <HelmetIntl
+        title={{ id: 'test', defaultMessage: 'Preview SMS' }}
+        description={{
+          id: 'test',
+          defaultMessage: 'Text message preview',
+        }}
+      />
+      <TextingHeader
+        headerMessage="Preview SMS message"
+        onClickGoBack={() => {
+          const url = `/admin/messaging/texting/${campaignId}/`;
+          clHistory.replace(url);
+        }}
+        showHorizontalRule
+      >
+        <ButtonContainer>
+          <Button
+            onClick={() => {
+              console.log('go back to the create screen');
+            }}
+            buttonStyle="secondary"
+            size="1"
+            text={'Edit'}
+          />
+          <Button
+            onClick={() => {
+              setShowConfirmationModal(true);
+            }}
```

Comment:
Normally you should get a warning for lambdas as a prop. This should go via a handler. Although I'm not sure if this is a real issue TBH.